shared/src/api/client/mainthreadAPI.ts

```diff
@@ -25,6 +27,13 @@ export const initMainThreadAPI = (
             .subscribe()
     )
 
+    // Workspace
+    // eslint-disable-next-line no-void
+    subscription.add(roots.subscribe(rs => void ext.syncRoots(rs || [])))
```

Comment:
```suggestion
    subscription.add(roots.subscribe(rs => {
        ext.syncRoots(rs || [])
    }))
```

Any reason to use `void` + eslint-disable versus just using a block?

Reply:
not really :)